src/QMCDrivers/WFOpt/QMCCostFunctionBatched.cpp

```diff
@@ -225,9 +225,25 @@ void compute_batch_parameters(int sample_size, int batch_size, int& num_batches,
   }
 }
 
-/** evaluate everything before optimization */
+//Blank function, can be removed after this function is removed in the base class
 void QMCCostFunctionBatched::checkConfigurations()
+{}
+
+/** evaluate everything before optimization */
+void QMCCostFunctionBatched::checkConfigurations(engineData& data)
```

Comment:
This method is huge and `data` is a super generic name that I would assume point to the major data structor or container to be operated on by the function. 

Here data is actually two raw pointers and a string that should be an enum class. There are at least 3 different method here that are branched around based on matches to that string and then unchecked uses of the raw pointers.